tls/s2n_recv.c

```diff
@@ -132,6 +132,7 @@ ssize_t s2n_recv(struct s2n_connection *conn, void *buf, ssize_t size, int *more
             if (r == -2) {
                 conn->closed = 1;
                 GUARD(s2n_connection_wipe(conn));
+		*more = 0;
```

Comment:
Must have been a tabs-vs-spaces issue.  There were no other tabs in the file so I changed it to spaces.